TemplePlus/python/python_object.cpp

```diff
@@ -2443,6 +2444,11 @@ static PyObject* PyObjHandle_Unwield(PyObject* obj, PyObject* args) {
 		es = EquipSlot::Invalid;
 	auto equipSlot = (EquipSlot)es;
 
+	if (item != objHndl::null && inventory.GetParent(item) == self->handle ) {
```

Comment:
item_unwield currently operates on a specific item slot. Passing in the item handle is redundant in this sense, simply adding an integer arg should be enough and IMO less confusing.